app/assets/javascripts/services/subscription_service.js

```diff
@@ -0,0 +1,9 @@
+ManageIQ.angular.app.service('subscriptionService', ['$timeout', function($timeout) {
+  this.subscribeToEventType = function(eventType, callback) {
+    ManageIQ.angular.rxSubject.subscribe(function(event) {
+      if (event.eventType === eventType) {
+        $timeout(callback(event.response));
```

Comment:
Ah, gotcha, yeah, `callback` is just a reference to a function, not a function that returns a function. Will fix this.